catalog/imaging/catalog-imaging-service-impl/src/main/java/com/connexta/alliance/imaging/chip/service/impl/GeoChipDescriptor.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) Connexta, LLC
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package com.connexta.alliance.imaging.chip.service.impl;
+
+import java.awt.Point;
+
+import com.vividsolutions.jts.geom.Coordinate;
+
+class GeoChipDescriptor {
```

Comment:
It isn't used outside the package and it isn't designed for reuse.  It's an implementation detail and I think it's better to keep things as private as possible.